.vscode/settings.json

```diff
@@ -6,9 +6,7 @@
     "search.exclude": {
         "out": true // set this to false to include "out" folder in search results
     },
-    "[typescript]": {
-        "editor.defaultFormatter": "esbenp.prettier-vscode"
-    },
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
Note that we now use prettier for all file types that it supports

Reply:
Have you confirmed that it's not trying to use prettier for python for example. "that it supports" it's not something I would rely on to work automatically.

Reply:
Yeah I just tried saving and it seems to be using black